common/variant/src/main/java/org/apache/spark/types/variant/VariantUtil.java

```diff
@@ -19,6 +19,7 @@
 
 import org.apache.spark.QueryContext;
 import org.apache.spark.SparkRuntimeException;
```

Comment:
`common/variant` still depends on `common/utils`(Scala) because this is a Scala class